pom.xml

```diff
@@ -178,7 +178,7 @@
 			<plugin>
 				<groupId>org.apache.maven.plugins</groupId>
 				<artifactId>maven-antrun-plugin</artifactId>
-				<version>3.1.0</version>
+				<version>3.3.0</version>
```

Comment:
i think you need to rebase your branch and then raise a PR, looks like old commits are also coming up and kee this 3.3.0 to 3.1